src/config.js

```diff
@@ -74,7 +74,13 @@ const config = {
         type: 'boolean',
         default: true,
         order: 10
-    }
+    },
+    'iconListStyle': {
+        description: 'Display only icons in update and create window (for more space).',
```

Comment:
Does it make sense to be more like: `'Display as *only icons* in create/update view (for more space).'`?